.github/workflows/regen-answers.yaml

```diff
@@ -5,9 +5,7 @@ on:
   schedule:
     - cron: "0 1 * * *"
 
-permissions:
-  contents: read
-  id-token: write # Needed to federate tokens.
```

Comment:
Just curious, why this change? Don't we need perms?

Reply:
the perms is set in the job level and not in the workflow

Reply:
those are the permissions to read the local repo and to use octo sts